tests/measurements/test_sample.py

```diff
@@ -453,76 +453,85 @@ def circuit(angle):
 
 
 @pytest.mark.jax
-@pytest.mark.parametrize("samples", (1, 10))
-def test_jitting_with_sampling_on_subset_of_wires(samples):
-    """Test case covering bug in Issue #3904.  Sampling should be jit-able
-    when sampling occurs on a subset of wires. The bug was occuring due an improperly
-    set shape method."""
-    import jax
+class TestJAXCompatibility:
```

Comment:
This diff looks weird because this was the 4th test in a row about JAX compatibility, so I put them all in a class together and moved the `@pytest.mark.jax` to the top. 

Indenting and moving `@pytest.mark.jax` is the only thing that changed aside from adding the test in lines 525-534, but it seems to really have confused GitHub.